iOS/DuckDuckGo/Subscription/UserScripts/SubscriptionPagesUseSubscriptionFeature.swift

```diff
@@ -351,17 +365,21 @@ final class DefaultSubscriptionPagesUseSubscriptionFeature: SubscriptionPagesUse
 
         setTransactionStatus(.polling)
 
+        var subscriptionParameters: [String: String]?
+
         // Free Trials Experiment Parameters & Pixels
-        var freeTrialParameters: [String: String]?
         if shouldPerformFreeTrialPostPurchaseActions {
-            freeTrialParameters = completeSubscriptionFreeTrialParameters
+            subscriptionParameters = completeSubscriptionFreeTrialParameters
             fireFreeTrialSubscriptionPurchasePixel(for: subscriptionSelection.id)
+        } else if let feExperiment = subscriptionSelection.experiment {
```

Comment:
I would prefer avoiding acronyms, maybe it would be more clear to make it just `experiment` or `frontendExperiment`?
(same comment for the V2 copy below)

Reply:
I actually agree with this, not sure why I still used one.  :)

Changing...

Reply:
Expanded acronym.